tests/golden/vshn/appcat/appcat/21_composition_vshn_postgres.yaml

```diff
@@ -95,6 +95,22 @@ spec:
               kind: SGInstanceProfile
               metadata: {}
               spec:
+                containers:
+                  cluster-controller:
+                    cpu: 100m
+                    memory: 256Mi
+                  envoy:
+                    cpu: 100m
+                    memory: 64Mi
+                  pgbouncer:
+                    cpu: 100m
+                    memory: 64Mi
+                  postgres-util:
+                    cpu: 100m
+                    memory: 256Mi
+                  prometheus-postgres-exporter:
+                    cpu: 100m
+                    memory: 256Mi
```

Comment:
Are we oke with these not adhering to the ratio and that we require the customer to do the math?

Reply:
Yes, the documentation that makes the customer aware of this was already approved and merged.